src/modules/general/apiLoadSettings.js

```diff
@@ -1,21 +1,25 @@
 import api from '../../utils/api'
 import { GENERAL_SETTINGS, NAMESPACE } from '../../constants/data-store'
+import { getAuthority } from '../getAuthority'
 
 /**
  * When general settings component mount, get namespace and keys from dataStore and return value
  */
 export const apiLoadGeneralSettings = async () => {
-    return api.getNamespaces().then(res => {
-        if (res.includes(NAMESPACE)) {
-            return api.getKeys(NAMESPACE).then(res => {
-                if (res.includes(GENERAL_SETTINGS)) {
-                    return api
-                        .getValue(NAMESPACE, GENERAL_SETTINGS)
-                        .then(res => {
-                            return res.value
-                        })
-                }
-            })
-        }
+    return getAuthority().then(hasAuthority => {
```

Comment:
It would be better not to n'est a bunch of callbacks here, this is very difficult to understand at a glance.  I would recommend splitting it out into well-named helper functions